env0/resource_team_project_assignment.go

```diff
@@ -59,14 +60,20 @@ func resourceTeamProjectAssignmentRead(ctx context.Context, d *schema.ResourceDa
 		return diag.Errorf("could not get TeamProjectAssignment: %v", err)
 	}
 
+	found := false
 	for _, assignment := range assignments {
 		if assignment.Id == id {
 			d.Set("project_id", assignment.ProjectId)
 			d.Set("team_id", assignment.TeamId)
 			d.Set("role", assignment.ProjectRole)
+			found = true
 			break
 		}
 	}
+	if !found {
+		d.SetId("")
+		return nil
```

Comment:
Seems like a redundant return statement

Reply:
Yes it is, will be fixed

Reply:
234334f5b27de1837aad58db525caff70bd002e0